node/src/main/scala/org/bitcoins/node/NeutrinoNode.scala

```diff
@@ -26,7 +26,7 @@ case class NeutrinoNode(
     nodeConfig: NodeAppConfig,
     chainConfig: ChainAppConfig,
     actorSystem: ActorSystem,
-    configPeersOverride: Vector[Peer] = Vector.empty)
+    paramPeers: Vector[Peer] = Vector.empty)
```

Comment:
So we don't override the config ones anymore, rather use both. This was one of the suggestion earlier so added in this.